Modules/_ctypes/_ctypes.c

```diff
@@ -2175,7 +2068,8 @@ PyCSimpleType_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
         }
     }
 
-    if (type == &PyCSimpleType_Type && fmt->setfunc_swapped && fmt->getfunc_swapped) {
+    ctypes_state *st = GLOBAL_STATE();
+    if (type == st->PyCSimpleType_Type && fmt->setfunc_swapped && fmt->getfunc_swapped) {
```

Comment:
Since we're touching this line, let's use the opportunity to apply PEP-7:

```suggestion
    if (type == st->PyCSimpleType_Type
        && fmt->setfunc_swapped
        && fmt->getfunc_swapped)
    {
```